src/layouts/Settings/Settings.tsx

```diff
@@ -37,6 +41,14 @@ import { PrivacySettings } from "./PrivacySettings"
 import { SocialMediaSettings } from "./SocialMediaSettings"
 
 export const Settings = (): JSX.Element => {
+  const growthbook = useGrowthBook<FeatureFlags>()
+  const fallbackWhitelist: PrivatisationWhitelist = { repos: [] }
+  const repoPrivatisationWhitelist =
+    growthbook?.getFeatureValue(
+      FEATURE_FLAGS.REPO_PRIVATISATION,
+      fallbackWhitelist
+    ) || fallbackWhitelist
```

Comment:
Hmm abit a bit confused here, why would growthbook be undefined ah? 
+ how do we reason about `fallbackWhiteList`? if this list does not contain anything, should this still live in production line ya?

Reply:
Return of useGrowthbook is this:

```
 useGrowthBook<FeatureFlags>(): GrowthBook<FeatureFlags> | undefined
```

It can be undefined because under the hood it uses a react context and growthbook if not yet initialised will return undefined

The fallbackWhitelist helps in 2 scenarios:
1. If the useGrowthbook call returns undefined
2. If growthbook is initialised but the fetching from growthbook fails

Hence, it should be there for defensiveness

Reply:
So moving forward we are going to maintain 2 sot arh? more in favour of just || [], and we expect growthbook to work most of the time, but if it doesnt we have a sane value to prevent app from crashing rather than maintaining 2 sot!

Reply:
Wait, how did you get 2 sot? There is still 1 sot on growthbook. The fallback value here is already the mechanism for preventing the app from crashing